flow-server/src/main/resources/vite.generated.ts

```diff
@@ -225,7 +225,8 @@ function statsExtracterPlugin(): PluginOption {
       const generatedImports = readFileSync(path.resolve(generatedFlowImportsFolder, "generated-flow-imports.js"), {encoding: 'utf-8'})
           .split("\n")
           .filter((line: string) => line.startsWith("import"))
-          .map((line: string) => line.substring(line.indexOf("'")+1, line.lastIndexOf("'")));
+          .map((line: string) => line.substring(line.indexOf("'")+1, line.lastIndexOf("'")))
```

Comment:
smallie :), but this line could be reformated like:
```suggestion
          .map((line: string) => line.substring(line.indexOf("'") + 1, line.lastIndexOf("'")))
```